pkg/instance/instance.go

```diff
@@ -191,6 +195,11 @@ func (i *Instance) AppsFS(appsType apps.AppType) afero.Fs {
 	panic(fmt.Errorf("Unknown application type %s", string(appsType)))
 }
 
+// DiskQuota returns the number of bytes allowed on the disk to the user.
+func (i *Instance) DiskQuota() (int64, error) {
+	return i.BytesDiskQuota, nil
```

Comment:
It implements an interface `DiskThresholder` which defines a `DiskQuota() (int64, error)`, allowing the possibility to have an error... maybe that is pretty useless for now...